assets/src/styles/campaigns/themes/_theme_antarctic.scss

```diff
@@ -266,4 +266,31 @@ body.theme-antarctic {
   .video-block {
     @include campaign_video($antarctic-gradient);
   }
+
```

Comment:
If we merge https://github.com/greenpeace/planet4-master-theme/pull/1012 first, maybe we can apply this already to these new styles, and use variables in Spreadsheet.scss instead of these theme specific files? Or do you think it's too early for that?

Reply:
Hmmm I don't think I follow, but I think we can keep the scope of this ticket as is and do incremental refactors as PRs of the refactor ticket.

Reply:
I agree with your point as far as refactors go. But in this case it's not a refactor but new code. So I was just wondering if we could avoid introducing any more campaign theme specific css rules if we already have the css variables code in place. Or is there something else that we don't have yet that blocks us from already doing that?

Reply:
Sorry, I thought you were talking about the `.video-block` code above this, because that was what Github was showing on the "Conversation" tab.

I guess we could add these color values somewhere in the JSON files and use CSS variables in Spreadsheet.scss.

Reply:
Nothing is blocking us, is just that this PR is from January 22nd, and the vars weren't there yet.

Reply:
@Inwerpsel Actually this needs this to happen too: https://github.com/greenpeace/planet4-master-theme/pull/1012/files